tests/plugins/test_amp_plugins.py

```diff
@@ -174,3 +175,68 @@ def test_amp_apex_ddp_spawn_fit(amp_level, tmpdir):
     assert isinstance(trainer.precision_plugin, ApexMixedPrecisionPlugin)
     model = BoringModel()
     trainer.fit(model)
+
+
+@RunIf(min_gpus=1, amp_native=True)
+def test_amp_precision_16_bfloat_disabled(tmpdir):
```

Comment:
this tests seems partially redundant given the very fist test in this file where we already have isinstance check. maybe worth merging